### DISCUSSION TITLE
Ubuntu WSL getting detected as Linux, not Ubuntu

### DISCUSSION THREAD

**Discussion by svenlowry-cgi** [JanDeDobbeleer/oh-my-posh#1646] [Troubleshoot] [answered]
I have the following segment in my profile:
`{
          "type": "os",
          "style": "powerline",
          "foreground": "#100e23",
          "background": "#cfcfc4",
          "background_templates": [
            "{{if .Root}}#ff3026{{else}}#cfcfc4{{end}}"
          ],
          "properties": {
            "prefix": " ",
            "postfix": "",
            "display_host": false,
            "wsl": "",
            "wsl_separator": ""
          }
        },`
On 1 machine, in my WSL install it shows the OS icon as the Ubuntu logo. On the other machine, it shows it as a Linux icon.
Both machines are running the same version of Windows 10 as a host, and the same Ubuntu 20.04.3 LTS as the WSL VM.

Any ideas why the OS module is returning as Linux rather than Ubuntu on one of them?

**Comment by lewis-yeung** [comment, marked as answer]
Possibly related: #1766. Try the latest release and delete the cache file `$POSH_PATH/omp.cache`. The WSL distro icon issue had been fixed after v7.43.1.

**Reply by JanDeDobbeleer** [reply]
You can also use `oh-my-posh cache clear` for that 😃

**Reply by svenlowry-cgi** [reply]
This has worked, thanks!
